src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimerEntryListViewViewModel.cs

```diff
@@ -12,9 +12,14 @@ public TimerEntryListViewViewModel()
         {
             this.WhenValueChanged(x => SelectedTab)
                 .Subscribe(value => Toggl.SetActiveTab(value));

```

Comment:
Can you please explain the diff in this file?
I cannot spot any actual difference in logic

Reply:
The event OnDisplayTimelineUI arrives later, after all the UI is created and until that time IsTimelineViewEnabled property is false. Reactive extension WhenAnyValue takes into account the initial value of IsTimelineViewEnabled property and resets the tab to List view. After that, even if IsTimelineViewEnabled=true arrives the tab is already reset.

Reply:
Thanks for the explanation.
Next time please follow the Pull Request Etiquette, to make it easier for everybody:
> Do not add unrelated changes - even minor code cleanup - to your pull request
> - Create a separate pull request instead
> - If possible, split larger pull requests into multiple ones